label_studio/data_manager/api.py

```diff
@@ -271,13 +275,18 @@ def get(self, request):
             else:
                 serializer = self.task_serializer_class(page, many=True, context=context)
             return self.get_paginated_response(serializer.data)
+
+        # TODO
         # all tasks
-        if project.evaluate_predictions_automatically:
-            evaluate_predictions(queryset.filter(predictions__isnull=True))
+        # if project.retrieve_predictions_automatically:
+        #     retrieve_predictions(queryset.filter(predictions__isnull=True))
```

Comment:
Similarly, we shouldn't merge commented out code here. What's the intent of this part of the change?